R/utils-shared.R

```diff
@@ -175,10 +175,21 @@ formatComments <- function(comments){
 ############ used in various places ############ 
 
 isEmpty <- function(val){
-  if(!is.null(val) && !is.na(val)){
-    result <- FALSE
+  if(is.null(val) || is.na(val)){
+    result <- TRUE
   } else {
+    result <- FALSE
+  }
+  return(result)
+}
+
+############ used in various places ############ 
+
+isEmptyOrBlank <- function(val){
+  if(is.null(val) || val==""){
```

Comment:
I honestly don't know if it needs to. Just was confused because I wasn't sure if isEmptyOrBlank should do everything that isEmpty does and more.